sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseDirectorySteps.java

```diff
@@ -478,39 +524,49 @@ public CaseDirectorySteps(
         (String checkboxDescription) -> {
           switch (checkboxDescription) {
             case ("Only cases without geo coordinates"):
+            case ("Nur F\u00E4lle ohne Geo-Koordinaten"):
               webDriverHelpers.clickOnWebElementBySelector(CASES_WITHOUT_GEO_COORDINATES_CHECKBOX);
               break;
             case ("Only cases without responsible officer"):
+            case ("Nur F\u00E4lle ohne verantwortlichen Beauftragten"):
               webDriverHelpers.clickOnWebElementBySelector(
                   CASES_WITHOUT_RESPONSIBLE_OFFICER_CHECKBOX);
               break;
             case ("Only cases with extended quarantine"):
+            case ("Nur F\u00E4lle mit verl\u00E4ngerter Isolation"):
               webDriverHelpers.clickOnWebElementBySelector(CASES_WITH_EXTENDED_QUARANTINE_CHECKBOX);
               break;
             case ("Only cases with reduced quarantine"):
+            case ("Nur F\u00E4lle mit verk\u00FCrzter Isolation"):
               webDriverHelpers.clickOnWebElementBySelector(CASES_WITH_REDUCED_QUARANTINE_CHECKBOX);
               break;
             case ("Help needed in quarantine"):
+            case ("Ma\u00DFnahmen zur Gew\u00E4hrleistung der Versorgung"):
               webDriverHelpers.clickOnWebElementBySelector(
                   CASES_HELP_NEEDED_IN_QUARANTINE_CHECKBOX);
               break;
             case ("Only cases with events"):
+            case ("Nur F\u00E4lle mit Ereignissen"):
               webDriverHelpers.clickOnWebElementBySelector(CASES_WITH_EVENTS_CHECKBOX);
               break;
             case ("Only cases from other instances"):
+            case ("Nur F\u00E4lle von anderen Instanzen"):
               webDriverHelpers.clickOnWebElementBySelector(CASES_FROM_OTHER_INSTANCES_CHECKBOX);
               break;
             case ("Only cases with reinfection"):
+            case ("Nur F\u00E4lle mit Reinfektion"):
               webDriverHelpers.clickOnWebElementBySelector(CASES_WITH_REINFECTION_CHECKBOX);
               break;
             case ("Include cases from other jurisdictions"):
               webDriverHelpers.clickOnWebElementBySelector(CASES_FROM_OTHER_JURISDICTIONS_CHECKBOX);
               break;
             case ("Only cases with fulfilled reference definition"):
+            case ("Nur F\u00E4lle mit erf\u00FCllter Referenzdefinition"):
               webDriverHelpers.clickOnWebElementBySelector(
                   CASES_WITH_FULFILLED_REFERENCE_DEFINITION_CHECKBOX);
               break;
             case ("Only port health cases without a facility"):
+            case ("Nur Einreisef\u00E4lle ohne zugewiesene Einrichtung"):
```

Comment:
so.. this will get messy, the method is too big, and I understand that there's no other way to handle it..
Can we create an enum where to store all this data, with the english and german translations, and pick them from there? so in this way, if another environment will come, we'll have a handle for it as well

Reply:
Tried to refactor it but switch didn't let me use created enum so it stays like that for now :)